addons/udes_stock/models/stock_picking.py

```diff
@@ -296,57 +332,74 @@ def _backorder_move_lines(self):
         self.ensure_one()
 
         moves = self.move_lines
-        mls = self.move_line_ids
+        # Only look at moving out incomplete moves from the picking
+        # TODO: This should probably be a method in stock_move
```

Comment:
Nit:
I personally don't love the idea of moving these into the main model simply because of readability but I do agree it's better than writing it again and again. Do you think you can implement this todo in stock.move? As I said, it's more of a nit, not a blocker.